armi/reactor/converters/uniformMesh.py

```diff
@@ -555,6 +584,11 @@ def applyStateToOriginal(self):
         .. impl:: Map select parameters from composites on the new mesh to the original mesh.
             :id: I_ARMI_UMC_PARAM_BACKWARD
             :implements: R_ARMI_UMC_PARAM_BACKWARD
+
+            To ensure that the parameters on the original Reactor are from the converter Reactor,
+            the first step is to clear the state on the original Reactor (see ``_clearStateOnReactor``).
```

Comment:
This should clarify that `_clearStateOnReactor` only clears the reactor parameters, not the block parameters.

Reply:
https://github.com/terrapower/armi/pull/1618/commits/4950969e069f68cd0fde98470ba33e0540ba98a9 let me know what you think